pkg/logevents/logevents.go

```diff
@@ -0,0 +1,98 @@
+/*
+* Copyright 2020 Armory, Inc.
+
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+
+*    http://www.apache.org/licenses/LICENSE-2.0
+
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+
+package logevents
+
+import (
+	"encoding/json"
+	"github.com/armory/dinghy/pkg/cache"
+	log "github.com/sirupsen/logrus"
+	"strconv"
+	"time"
+)
+
+type LogEventsClient interface {
+	GetLogEvents() ([]LogEvent, error)
+	SaveLogEvent(logEvent LogEvent) error
+}
+
+type LogEventRedisClient struct {
+	RedisClient *cache.RedisCache
+}
+
+type LogEvent struct {
+	Org 		string
+	Repo 		string
+	Files		[]string
+	Message		string
```

Comment:
Is there a preference for the pascal case or can we change to the camel case? 
It's indifferent to the UI, but all the current responses are camel case.

Reply:
This are in Capital to make them public, added custom tags so everything is in lowercase when converted to json.